stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/MessagesActivity.kt

```diff
@@ -139,26 +142,45 @@ class MessagesActivity : AppCompatActivity() {
                 )
             }
 
```

Comment:
A bit of duplication here

Reply:
Given that our screen components and the Activities for sample are not _as_ important, we can afford it